sdk/tables/azure-data-tables/inc/azure/data/tables/account_sas_builder.hpp

```diff
@@ -123,54 +106,19 @@ namespace Azure { namespace Data { namespace Tables { namespace Sas {
     Delete = 4,

```

Comment:
Are these numeric constants defined somewhere? If they are, the definitions should be referenced somewhere in the code.

If they are not, these should be updated to start at `2^0` rather than `2^2`.